lib/components/artist/__tests__/header-tests.js

```diff
@@ -0,0 +1,21 @@
+import 'react-native'
+
+import React from 'react'
+import renderer from 'react-test-renderer'
+
+import Header from '../header'
+
+jest.mock('../../spinner.js', () => 'ARSpinner')
+
+it('renders properly', () => {
+  const artist = {
+      name: 'Marcel Duchamp',
+      nationality: 'French',
+      birthday: '11/17/1992',
+      counts: {
+        follows: 22,
```

Comment:
I definitely agree that we should test the follow button for those cases. It could use some logic tests but for now I'm just doing snapshots.